Survey/Sources/Data/NetworkAPI/Core/AuthenNetworkAPI.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  AuthenNetworkAPI.swift
+//  Survey
+//
+//  Created by Khanh on 17/08/2022.
+//  Copyright © 2022 Nimble. All rights reserved.
+//
+
+import Alamofire
+import Combine
+
+final class AuthenNetworkAPI: NetworkAPIProtocol {
```

Comment:
I prefer `AuthNetworkAPI` or `AuthenticationNetworkAPI`. What do you think?
```suggestion
final class AuthenNetworkAPI: NetworkAPIProtocol {
```